content/docs/command-reference/exp/index.md

```diff
@@ -0,0 +1,65 @@
+# experiments
+
+⚠️ This feature is only available in DVC 2.0 ⚠️
+
+A set of commands to generate and manage <abbr>experiment</abbr>:
+[run](/doc/command-reference/exp/run), [show](/doc/command-reference/exp/show),
+[diff](/doc/command-reference/exp/diff),
+[apply](/doc/command-reference/exp/apply),
+[branch](/doc/command-reference/exp/branch),
+[resume](/doc/command-reference/exp/resume),
+[gc](/doc/command-reference/exp/gc), [list](/doc/command-reference/exp/list),
+[push](/doc/command-reference/exp/list), and
+[pull](/doc/command-reference/exp/pull).
+
+> Aliased to `dvc exp`.
+
+## Synopsis
+
+```usage
+usage: dvc experiments [-h] [-q | -v]
+       {show,apply,diff,run,resume,res,gc,branch,list,push,pull} ...
+
+positional arguments:
+  COMMAND
+    show          Print experiments.
+    apply         Apply the changes from an experiment to your workspace.
+    diff          Show changes between experiments in the DVC repository.
+    run           Reproduce complete or partial experiment pipelines.
+    resume (res)  Resume checkpoint experiments.
+    gc            Garbage collect unneeded experiments.
+    branch        Promote an experiment to a Git branch.
+    list          List local and remote experiments.
+    push          Push a local experiment to a Git remote.
+    pull          Pull an experiment from a Git remote.
+```
+
+## Description
+
+`dvc exp` subcommands provide specialized ways to create and manage data science
+experiments. The basic workflow goes like this:
+
+- Modify <abbr>dependencies</abbr> (e.g. input data or source code),
+  <abbr>hyperparameters</abbr>, or commands (`cmd` field of `dvc.yaml`) of
```

Comment:
Yes but isn't `dvc exp` pretty much designed for DS/ML?

Reply:
p.s. the tooltip may help:

![image](https://user-images.githubusercontent.com/1477535/108668865-95665380-74a1-11eb-8e81-3ccf16e57ade.png)

Reply:
> Yes but isn't `dvc exp` pretty much designed for DS/ML?

Yup, and I understand the preference for `hyperparameters` because of that. I thought `parameters` might be more clear since this is a technical reference, not a user guide, and `parameters` is the term used elsewhere to describe the full scope of what can be modified, rather than expected usage. Also, looking at the getting started example, experiments are used to change the upstream featurization parameters, not only the model hyperparameters.

That being said, the tooltip does show that general description of parameters, so if you think it's more helpful to use `hyperparemeters`, I'm fine with it.

Reply:
Changed, PTAL @dberenbaum